src/main/java/org/jabref/gui/linkedfile/DeleteFileAction.java

```diff
@@ -116,7 +116,32 @@ private void deleteFiles(List<LinkedFileViewModel> toBeDeleted, boolean deleteFr
      */
     public void deleteFileFromDisk(LinkedFileViewModel fileViewModel) {
         LinkedFile linkedFile = fileViewModel.getFile();
+        deleteFileHelper(dialogService, preferences, databaseContext, linkedFile);
+    }
+
+    /**
+     * Deletes files from disk without asking the user for confirmation, for a list of files
```

Comment:
Wrong text - The `@return` says something else. - Either adapt text - or delete text